commands/utils/consts.go

```diff
@@ -72,4 +72,12 @@ const (
 
 	// The 'GITHUB_ACTIONS' environment variable exists when the CI is GitHub Actions
 	GitHubActionsEnv = "GITHUB_ACTIONS"
+
+	// Amount of days in which we want to rescan a repo even it had no changes to the code.
```

Comment:
// Amount of days in which we want to rescan a repo even it had no changes to the code.
-->
// When Frogbot periodically scans repositories, it skips scanning repositories for which the latest commit has already been scanned, unless the latest commit has been scanned more then 'SkipRepoScanDays' days ago.